openedx/core/djangoapps/edx_global_analytics/utils/cache_utils.py

```diff
@@ -52,6 +53,13 @@ def get_query_result(query_type, activity_period):
             ).annotate(count=Count('country')).values_list('country', 'count')
         )
 
+    if query_type == 'generated_certificates':
+        generated_certificates = GeneratedCertificate.objects.extra(select={'date': 'date( created_date )'}).values(
+            'date'
+        ).annotate(count=Count('pk'))
+
+        return {str(c.get('date', '')): c.get('count', 0) for c in generated_certificates}
```

Comment:
I would use `strftime` if generated_certificates is a datetime.datetime object instead of straightforward converting date to str.